utils/gen-snapshot-tests.js

```diff
@@ -27,12 +27,13 @@ const ignored = [
     "**/*.d.ts",
 ];
 
-function generateTestFile(root, examples, componentFileMap) {
+function generateTestFile(root, {examples, componentFileMap}) {
     const lines = [
         "// @flow",
         "// This file is auto-generated by gen-snapshot-tests.js",
         "// Do not edit this file.  To make changes to these snapshot tests",
-        `// edit ${root}/docs.md and run \`npm run gen-snapshot-tests\`.`,
+        `// edit ${root}/docs.md and other documentation files in the package,`,
+        "// then run `npm run gen-snapshot-tests`.",
```

Comment:
Updated the comment to reflect what now happens (should probably update this to yarn too, but I don't know that I want to do all that rebasing and such. Maybe as a future edit).